### PR TITLE
Update meetup 4

### PR DESCRIPTION
**Pull request by yishus**

### CODE REVIEW

File: content/meetups/4.md [hunk 1]
```diff
@@ -14,16 +14,27 @@ talks:
 
 ## ⚛️ React updates ⚛️
 
-### ⚛️ [React 16.11.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#16110-october-22-2019)
+### ️⚛️ [React Conf](https://www.youtube.com/playlist?list=PLPxbbTqCLbGHPxZpw4xj_Wwg8-fdNxJRh)
```
Comment (thchia) [inline content/meetups/4.md]: Got some icon or something here? Square box

Comment (yishus) [reply content/meetups/4.md]: it's the chemical reaction emoji. how does it look like from your side?

Comment (thchia) [reply content/meetups/4.md]: 6710EACB-E48C-48BA-B4E4-624551687E11

Comment (yishus) [reply content/meetups/4.md]: ohhh thanks for catching! fixed 🤞

**Comment by wgao19** [comment]
watttttt plz just merge next time 😅

**Pull request merged**